content/how-to/rackspace-fabric/rackspace-fabric-faq/index.md

```diff
@@ -0,0 +1,71 @@
+---
+permalink: "rackspace-fabric-faq"
+audit_date: '2022-03-17'
+type: article
+title: "Rackspace Fabric FAQ"
+created_date: '2022-03-17'
+created_by: Asmita Nakwa
+last_modified_date: '2022-03-17'
+last_modified_by: Asmita
+product: Rackspace Fabric
+product_url: rackspace-fabric
+---
+
+### General
+{{<accordion title="How can I reset the password of my Email?" col="in" href="accordion1">}}
+
+If this is cloud account admin, they can review the following link (https://account.rackspace.com/event-history). It should help them understand any recent changes/access to their account.
+{{</accordion>}}
+
+{{<accordion title="How can I return back to my previous ticketing application?" col="in" href="accordion2">}}
+
+The customer needs to add the firewall as a device and the **Add VPN User** template will be visible in common requests.
+{{</accordion>}}
+
+{{<accordion title="I am not able to login to MyRack portal. I have been asked for MFA verification though I have not configured MFA yet." col="in" href="accordion3">}}
+
+MFA is mandatory for all customers but should allow customer to configure it. In case to remove the MFA please contact identity team.
+{{</accordion>}}
+
+{{<accordion title="I am not getting email notification via My Rackportal for incidents pertaining to Alert Logic." col="in" href="accordion4">}}
+
+The customer will get notification only if the device in question is opted in.{{</accordion>}}
```

Comment:
Do we have anywhere to link them for setting up or opting in?
Can we explain how they can fix and get email notifications working?
Khazi Pasha or Kshitij Mehta might be able to assist you with this answer if you don't have anyone to answer it.

Reply:
index.md file for notification preference will be created and linked to the answer. The admin (customer side) can check the preference whether the email notification is enabled or disabled for that particular device.